packages/server/src/plugin/traceTreeBuilder.ts

```diff
@@ -156,11 +152,21 @@ export class TraceTreeBuilder {
       if (specificNode) {
         node = specificNode;
       } else {
-        this.logger.warn(
-          `Could not find node with path ${path.join(
-            '.',
-          )}; defaulting to put errors on root node.`,
+        let nodePtr: Trace.INode = node;
```

Comment:
What's up with all the codecov warnings? There is a new test, no?

Reply:
Yeah, that's odd...the test wouldn't be passing if this code didn't run 🤔

Reply:
Seems to have resolved itself